applications/nrf_desktop/doc/main.rst

```diff
@@ -0,0 +1,30 @@
+.. _nrf_desktop_main:
+
+Main module
+###########
+
+The main module is the entry point of the application.
+
+
+Module Events
+*************
+
++------------------+-------------+--------------------------+------------------------+------------------+
+| Source Module    | Input Event | This Module              | Output Event           | Sink Module      |
++==================+=============+==========================+========================+==================+
+|                  |             |  :ref:`nrf_desktop_main` | ``module_state_event`` |                  |
```

Comment:
@MarekPieta - discussed offline. I think this is required for completeness but what we can do is to create an intermediate page that would hide source and sink modules for event where the number of these is large.

Reply:
Hi @greg-fer , it seems inconsistent at the moment so we need to decide on something and fix it everywhere. @MarekPieta will generate these tables using the script so we don't have to manually maintain them. Going back to the your original comment I would prefer to have a link as then the format would be the same as for source and sink.

Reply:
I'd really prefer not to link to the page you're on. It's very confusing. If the script cannot handle it, then I'd prefer to discuss this more.

Reply:
The script can decorate module name under `This module` column without `ref`.
In case the module appears also as one of Source/Sink modules it will be decorated with `ref` (for consistency with other modules in the column).
Will this be ok for you ?